README.rst

```diff
@@ -134,7 +134,11 @@ From the ``ckanext-showcase`` directory::
     paster showcase migrate -c {path to production.ini}
 
 Note that each Related Item must have a unique title before migration can
-proceed.
+proceed. If you prefer resolving duplicates as showcases, you can use the force
+option to migrate them anyways. Duplicate Relations will be created as
```

Comment:
Can I suggest this text:

In some cases it may be easier to resolve duplicates after Related Items have been migrated to Showcases. In that case you can use the `--allow-duplicates` flag. Showcases with duplicate titles will be created with the title `'duplicate_' + original_related_title + '_' + hash`.